docker-compose.yml

```
@@ -0,0 +1,14 @@
+﻿version: "3"
+
+services:
+	app:
+    build: .
+    container_name: bot
+    command: python app.py
+    ports:
+      - 80:80
+    volumes:
+      - ./:/app
+		
+		env_file:
+			.env.exapmle
```

```suggestion
version: "3"

services:
  app:
    build: .
      container_name: bot
      command: python app.py
      volumes:
        - ./:/app
      env_file:
        .env
```
вот так должно быть, а то у тебя что-то с отступами

---

@elmiringos `python app.py` запускает и бота и asyncio?

---

@elmiringos у тебя еще поднимается здесь же redis и MongoDB, их надо отдельными сервисами поднять
вроде бы в статье на хабре которую я тебе кидал это есть